app/theme/client/imports/components/sidebar/sidebar-header.css

```diff
@@ -149,3 +144,69 @@
 		}
 	}
 }
+
+@media (width <= 500px) {
+	.sidebar__header {
+		padding: var(--sidebar-small-default-padding);
+		
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5491030)